packages/ag-charts-community/src/chart/label.ts

```diff
@@ -146,5 +146,7 @@ export function calculateLabelBBox(
     return {
         point: { x, y, size: 0 },
         label: { width, height, text },
+        marker: undefined,
+        placement: 'bottom',
```

Comment:
Is this default consistent with all existing cases - i.e. is there any risk this is a breaking change?

Reply:
I've managed to untangle these types. It never needed `size` either